### DISCUSSION TITLE
ScriptHookVDotNet "Nightly" versions not compatible with .vb scripts?

### DISCUSSION THREAD

**Discussion by JAM102970** [scripthookvdotnet/scripthookvdotnet#1537] [Q&A]
Does anybody knows how to get .vb scripts to work with SHVDN Nightly versions? I know by adding .2 to .cs scripts works but, I can't get .vb scripts to work with it.  If I install a .vb script into my gta 5 game it just crashes the game right off.  Any help or suggestions would be greatly appreciated.

**Comment by kagikn** [comment]
Can you show us any .vb scripts that can reproduce the issue, so we can make sure the issue is responsible to SHVDN?

**Reply by JAM102970** [reply]
Thanks for your quick response....and you're the guy I actually wanted to ask this question to, basically I have a single .vb script from a mod that you can use to either shorten or lengthen either days or nights....this is an awesome mod I always had installed in previous patches when SHVDN v3.6.0 was still working.   I like this mod because I can lengthen the days and shorten the nights, I hate playing when nights take too long to phase away so is there anyway, currently, with the "nightly" versions to make this mod work again?  Here's the link to that specific mod so you can familiarize with it >  https://www.gta5-mods.com/scripts/2x-longer-days-net-mod

**Reply by kagikn** [reply]
Sorry, but no, I couldn't repro the issue with the mod 2x Longer Days you told me and [v3.7.0-nightly.7](https://github.com/scripthookvdotnet/scripthookvdotnet-nightly/releases/tag/v3.7.0-nightly.7) on the condition that the .vb script is named `LongerDays.2.vb`. It could be other mods you've installed that are causing your game crashing issue.

**Reply by JAM102970** [reply]
I'm sorry I did not know I had to also add the .2 in front of .vb scripts.   I thought that only applied to .cs scripts.  I'll add .2 to that specific .vb script and I'll test it in game and see what happens.
